test/e2e/framework/managed.go

```diff
@@ -120,12 +120,14 @@ func (f *ManagedFramework) TestNamespaceName() string {
 func (f *ManagedFramework) SetUpControllerFixture(typeConfig typeconfig.Interface) {
 	// TODO(marun) check TestContext.InMemoryControllers before setting up controller fixture
 	kubeConfig := fedFixture.KubeApi.NewConfig(f.logger)
-	fixture := framework.NewSyncControllerFixture(f.logger, typeConfig, kubeConfig, fedFixture.SystemNamespace, fedFixture.SystemNamespace)
+	ns := fedFixture.SystemNamespace
```

Comment:
How come we're specifying the `SystemNamespace` here and in `SetUpServiceDNSControllerFixture` below for managed E2E, but in `initCrudTest` we use `metav1.NamespaceAll` for integration?